src/rules/value-keyword-case/index.js

```diff
@@ -0,0 +1,66 @@
+import valueParser from "postcss-value-parser"
+import {
+  cssWordIsVariable,
+  declarationValueIndexOffset,
+  report,
+  ruleMessages,
+  validateOptions,
+} from "../../utils"
+
+export const ruleName = "value-keyword-case"
+
+export const messages = ruleMessages(ruleName, {
+  expected: (actual, expected) => `Expected "${actual}" to be "${expected}"`,
+})
+
+const ignoredCharacters = new Set([
```

Comment:
Please just add a comment here explaining why these exist. Something like "Operators are interpreted as "words" by the value parser, so we want to make sure to ignore them."